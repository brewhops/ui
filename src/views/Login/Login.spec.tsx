import React from 'react';
import { Login } from './Login';
import { shallow } from 'enzyme';

describe('src/pages/Login/Login.tsx', () => {
  it('renders the Login page', () => {
    expect(shallow(<Login/>)).toMatchSnapshot();
  })
});