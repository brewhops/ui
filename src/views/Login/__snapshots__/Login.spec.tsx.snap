// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/Login/Login.tsx renders the Login page 1`] = `ShallowWrapper {}`;
